Also here???
```
2025-01-28 18:24:07,314 [   ERROR] Saving crash info to /OUTPUTS/BIDS/sub-200007109/log/20250128-074902_43f72bf2-3767-424b-9b2e-935cf59bc70b/crash-20250128-182407-duettwe-_midthickness1-52d20e50-0287-4f56-9116-e249db897e61.txt
Traceback (most recent call last):
  File "/opt/conda/envs/fmriprep/lib/python3.11/site-packages/nipype/pipeline/plugins/multiproc.py", line 67, in run_node
    result["result"] = node.run(updatehash=updatehash)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/fmriprep/lib/python3.11/site-packages/nipype/pipeline/engine/nodes.py", line 527, in run
    result = self._run_interface(execute=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/fmriprep/lib/python3.11/site-packages/nipype/pipeline/engine/nodes.py", line 645, in _run_interface
    return self._run_command(execute)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/fmriprep/lib/python3.11/site-packages/nipype/pipeline/engine/nodes.py", line 771, in _run_command
    raise NodeExecutionError(msg)
nipype.pipeline.engine.nodes.NodeExecutionError: Exception raised while executing Node _midthickness1.

Cmdline:
	mris_expand -pial /tmp/HeckersLEV-x-2-000071-09-x-242488-x-test-074353-fmriprep-HeckersLEV-freesurfer_v24-x-426dab9c.i3fyPTNJK/work/fmriprep_24_1_wf/sub_200007109_wf/anat_fit_wf/surface_recon_wf/midthickness/mapflow/_midthickness1/lh.pial -thickness -thickness_name /tmp/HeckersLEV-x-2-000071-09-x-242488-x-test-074353-fmriprep-HeckersLEV-freesurfer_v24-x-426dab9c.i3fyPTNJK/work/fmriprep_24_1_wf/sub_200007109_wf/anat_fit_wf/surface_recon_wf/midthickness/mapflow/_midthickness1/lh.thickness /tmp/HeckersLEV-x-2-000071-09-x-242488-x-test-074353-fmriprep-HeckersLEV-freesurfer_v24-x-426dab9c.i3fyPTNJK/work/fmriprep_24_1_wf/sub_200007109_wf/anat_fit_wf/surface_recon_wf/midthickness/mapflow/_midthickness1/lh.white 0.5 /tmp/HeckersLEV-x-2-000071-09-x-242488-x-test-074353-fmriprep-HeckersLEV-freesurfer_v24-x-426dab9c.i3fyPTNJK/work/fmriprep_24_1_wf/sub_200007109_wf/anat_fit_wf/surface_recon_wf/midthickness/mapflow/_midthickness1/lh.midthickness
Stdout:
	reading pial surface from /tmp/HeckersLEV-x-2-000071-09-x-242488-x-test-074353-fmriprep-HeckersLEV-freesurfer_v24-x-426dab9c.i3fyPTNJK/work/fmriprep_24_1_wf/sub_200007109_wf/anat_fit_wf/surface_recon_wf/midthickness/mapflow/_midthickness1/lh.pial
	using distance as a % of thickness
	using thickness file /tmp/HeckersLEV-x-2-000071-09-x-242488-x-test-074353-fmriprep-HeckersLEV-freesurfer_v24-x-426dab9c.i3fyPTNJK/work/fmriprep_24_1_wf/sub_200007109_wf/anat_fit_wf/surface_recon_wf/midthickness/mapflow/_midthickness1/lh.thickness
	expanding surface /tmp/HeckersLEV-x-2-000071-09-x-242488-x-test-074353-fmriprep-HeckersLEV-freesurfer_v24-x-426dab9c.i3fyPTNJK/work/fmriprep_24_1_wf/sub_200007109_wf/anat_fit_wf/surface_recon_wf/midthickness/mapflow/_midthickness1/lh.white by 50.0% of thickness and writing it to /tmp/HeckersLEV-x-2-000071-09-x-242488-x-test-074353-fmriprep-HeckersLEV-freesurfer_v24-x-426dab9c.i3fyPTNJK/work/fmriprep_24_1_wf/sub_200007109_wf/anat_fit_wf/surface_recon_wf/midthickness/mapflow/_midthickness1/lh.midthickness
	reading thickness...
Stderr:
	*** buffer overflow detected ***: terminated
	Aborted
Traceback:
	RuntimeError: subprocess exited with code 134.

250128-18:24:09,305 nipype.workflow ERROR:
	 could not run node: fmriprep_24_1_wf.sub_200007109_wf.anat_fit_wf.surface_recon_wf.midthickness
2025-01-28 18:24:09,305 [   ERROR] could not run node: fmriprep_24_1_wf.sub_200007109_wf.anat_fit_wf.surface_recon_wf.midthickness
250128-18:24:09,306 nipype.workflow ERROR:
	 could not run node: fmriprep_24_1_wf.sub_200007109_wf.anat_fit_wf.surface_recon_wf.midthickness
2025-01-28 18:24:09,306 [   ERROR] could not run node: fmriprep_24_1_wf.sub_200007109_wf.anat_fit_wf.surface_recon_wf.midthickness
250128-18:24:09,507 nipype.workflow CRITICAL:
	 fMRIPrep failed: 2 raised. Re-raising first.
2025-01-28 18:24:09,507 [CRITICAL] fMRIPrep failed: 2 raised. Re-raising first.
ERROR:missing required output:HTML
JOB_FAILED
DONE!
```

--- a/processors/fmriprep-HeckersLEV-freesurfer_v24.1.1.yaml
+++ b/processors/fmriprep-HeckersLEV-freesurfer_v24.1.1.yaml
@@ -68,6 +68,7 @@ command:
   type: singularity_run
   container: fmriprep
   args: >-
+    --work-dir /tmp
     --notrack
     --output-spaces MNI152NLin2009cAsym
     --ignore fieldmaps slicetiming

